src/main/java/org/generationcp/middleware/dao/audit/germplasm/GermplasmBasicDetailsAuditDAOQuery.java

```diff
@@ -0,0 +1,48 @@
+package org.generationcp.middleware.dao.audit.germplasm;
+
+import org.generationcp.middleware.dao.audit.AuditConstants;
+
+class GermplasmBasicDetailsAuditDAOQuery {
+
+	static final String LOCATION_NAME_ALIAS = "locationName";
+	static final String CREATION_DATE_ALIAS = "creationDate";
+
+	static final String LOCATION_CHANGED_ALIAS = "locationChanged";
+	static final String CREATION_DATE_CHANGED_ALIAS = "creationDateChanged";
+
+	private final static String BASE_QUERY = "SELECT "
```

Comment:
not sure how we will handle it in the future but I think we should use LEFT JOIN instead of INNER JOIN for the cases where the references where deleted. The problem with location and breeding methods is that we are showing the names and not the ids, so if the reference was deleted I dont know what we will show. We can leave it as it is now and continue the discussion and see how we can solve it 
What do you think @lik-83